common/data/src/main/java/org/thingsboard/server/common/data/kv/DeleteTsKvQuery.java

```diff
@@ -19,4 +19,6 @@ public interface DeleteTsKvQuery extends TsKvQuery {
 
     Boolean getRewriteLatestIfDeleted();
 
+    Boolean getDeleteLatest();
```

Comment:
Is it possible to set deleteLatest = false and rewriteLatestIfDeleted to true? What does 'If Deleted' mean?

Reply:
`        if (query.getDeleteLatest()) {
            futures.add(timeseriesLatestDao.removeLatest(tenantId, entityId, query));
        }`

Reply:
We will delete the latest value only if deleteLatest = true and delete period of time (startTs, endTs) is includes LatestTelemetry ts